README.md

```diff
@@ -18,3 +18,61 @@ _yarn_:
 ```bash
 yarn add @vesselapi/react-vessel-link
 ```
+
+### Usage
+The `vessel-link` modal can be triggered using two methods:
+1. Through the `VesselConnectButton` component.
+2. Through the `useVesselLink` hook.
+
+Here are some example uses of each:
+```jsx
+function App() {
+  return (
+    <VesselConnectButton
+      linkToken={linkToken}
+      onSuccess={(tempToken) => console.log('temp token: ', tempToken)}
+      onClose={() => console.log('closed')}
+      onLoad={() => console.log('loaded')}
+    >
+      Connect your CRM!
+    </VesselConnectButton>
+   )
+}
+```
+```js
+function App() {
+  const { open } = useVesselLink({ 
+    linkToken, 
+    onSuccess: (tempToken) => console.log('temp token: ', tempToken),
+    onClose: () => console.log('closed'),
+    onLoad: () => console.log('loaded')
+  })
+  
+  return <button onClick={() => open()}>Connect your CRM!</button>
+}
+```
+
+A common pattern is to embed each of the Vessel supported integrations directly into your app instead of using the selection modal. This allows for an additional level of white-labeling and will result in the modal opening directly to the given integration's authentication flow. Currently, this is only supported when using the `useVesselLink` hook.
```

Comment:
Start with the following? This way it's easier to skim and know what thi section is about?

If you would like to directly show the authentication for a given integration without letting your user's choose, ...